core/templates/dev/head/pages/exploration-editor-page/improvements-tab/feedback-improvement-task/feedback-improvement-task.directive.ts

```diff
@@ -65,6 +65,17 @@ angular.module('oppia').directive('feedbackImprovementTask', [
             return DateTimeFormatService.getLocaleAbbreviatedDatetimeString(
               $scope.getLatestMessage().updatedOn);
           };
+
+          $scope.wasUpdatedAtMost24HoursAgo = function() {
+            var feedbackDateOrTime =
```

Comment:
Do:

```
var feedbackDateOrTime = (
  DateTimeFormatService.getLocaleAbbreviatedDatetimeString(
    $scope.getLatestMessage().updatedOn));
```

Note the indentation.

Reply:
In the latest commit, I followed same indentation for lines over max length on Sublime but seems different in **files changed** on github. Is there automated way of fixing like using **--fix** with **git push**.

Reply:
No, you will need to make fixes manually.